### DISCUSSION TITLE
Error while installing

### DISCUSSION THREAD

**Discussion by fabisento** [seleniumbase/SeleniumBase#2425] [General]
Hi all,
 
quite simple and basic question, but are there special requirements for installing sbase apart from python?
 
After executing the command, all packages are downloaded but at the end i get this error:

ERROR: To modify pip, please run the following command:
C:\Users\Peter Max\AppData\Local\Programs\Python\Python311\python.exe -m pip install -e.
 
Hope someone can help

**Comment by mdmintz** [comment]
Likely an issue with your virtual environment. That Python error message is helpful in that it tells you what to do. There’s also more info on StackOverflow: https://stackoverflow.com/a/65985263/7058266

**Comment by fabisento** [comment]
Thank you very much, amigo!
The workaround of running the setup script via cmd from the downloaded zip file worked for me here
